docs/0010-differential-n-tuple-omit-prefix-storage-layout.md

```diff
@@ -0,0 +1,138 @@
+# OCFL Community Extension 0010: Differential N-Tuple Omit Prefix Storage Layout
+
+  * **Extension Name:** 0010-differential-n-tuple-omit-prefix-storage-layout
+  * **Authors:** Mike Giarlo
+  * **Minimum OCFL Version:** 1.1
+  * **OCFL Community Extensions Version:** 1.0
+  * **Obsoletes:** n/a
+  * **Obsoleted by:** n/a
+
+## Overview
+
+This OCFL storage root extension describes a storage layout combining a pairtree-inspired root directory structure intended to support identifiers with differential tuple sizes. For identifiers such as the [DRUID ("Digital Resource Unique IDentifier")](https://sdr.library.stanford.edu/documentation/purls-dois-and-orcid-ids), n-tuples in the identifier are of different sizes. For example, for the identifier `bc123df5678`, one would expect four tuples: `bc`, `123`, `df`, and `5678`.
+
+The OCFL object identifiers may contain prefixes which may be removed in the mapping to directory names. The OCFL object identifier prefix is defined as all characters before and including a configurable delimiter. The extension includes an option to include the full identifier as a leaf node.
+
+This layout is heavily based on the 0007-n-tuple-omit-prefix-storage-layout extension, adding the ability to handle tuples of different sizes.
```

Comment:
Should link [0007-n-tuple-omit-prefix-storage-layout](https://ocfl.github.io/extensions/0007-n-tuple-omit-prefix-storage-layout) . Stylistically I also suggest deleting "heavily" because "based" on its own is perhaps clearer and stronger.